slf4j-bridge/src/main/scala/org/slf4j/impl/ZioLoggerFactory.scala

```diff
@@ -43,7 +43,13 @@ class ZioLoggerFactory extends ILoggerFactory {
   private[impl] def run(f: ZIO[Any, Nothing, Any]): Unit =
     if (runtime != null) {
       zio.Unsafe.unsafe { implicit u =>
-        runtime.unsafe.run(f)
+        runtime.unsafe.run {
+          val fiberRefs = Fiber.currentFiber().map(_.asInstanceOf[Fiber.Runtime[_, _]].unsafe.getFiberRefs())
```

Comment:
You are within the ZIO name space so you could just use `_currentFiber` here to avoid the need for the cast. I will also look at specializing the return type of `currentFiber`.

Reply:
i changed it slf4j v2 bridge to use `_currentFiber`, as it is in zio namespace